Samsung/LPVS
# feat: History page(Add dropdown function)

# Description

"This Pull Request is linked to Issue #221. I have developed a feature that allows users to adjust the quantity of scanned data using a dropdown method.
Fixes #221

## Type of change



# How Has This Been Tested?

1. Update environment variables of docker-compose.yml, src/main/resources/application.properties, frontend/.env file.

2. Build the lpvs container : docker compose build

3. Run containers: docker compose up

4. See the lpvs home page with any browser: http://localhost:7896


![image](https://github.com/Samsung/LPVS/assets/111037279/4b5c8377-a5be-479f-8e0b-7d2c7444f969)


**Test Configuration**:
* Java: v11
* LPVS Release: v1.1.0

# Checklist:
